### ISSUE TITLE
Is the model being trained on both train and test set?

### ISSUE THREAD

**Issue by tomyoung903** [KAIST-AILab/NeuralPipeline_DSTC8#8]
Hi,

The total_v4.json file contains both the training and test set, and valid_v4.json contains just the valid set.

I see that total_v4.json is used for training as a whole.

Is that an error please?

Thank you.

**Comment by Victorwz** [comment]
I think it is because the authors only release the code for evaluating their model on Convlab. The test set of Convlab is not released and different from that of MultiWOZ. Therefore, the author can train their model both on the training set and test set of MultiWOZ.
